src/query-store.ts

```diff
@@ -570,6 +570,25 @@ export const useQueryCache = /* @__PURE__ */ defineStore(QUERY_STORE_ID, ({ acti
     },
   )
 
+  /**
+   * Preloads a query. Can only be called when including options.
+   * If provided `queryKey` already exists preload will refetch only if data is stale
+   */
+  const prefetch = action(
+    <TResult = unknown, TError = ErrorDefault>(
+      opts: UseQueryOptions<TResult, TError>,
+    ): void => {
+      const options: UseQueryOptionsWithDefaults<TResult, TError> = {
+        ...optionDefaults,
+        ...opts,
+      }
+
+      const entry = ensure<TResult, TError>(options)
+
+      refresh(entry).then(() => undefined).catch(() => undefined)
+    },
+  )
```

Comment:
Let's:

- make the function a one liner
- remove `action` because this won't be called within the store
- return the refresh call (implicit in a one-liner)
- replace the two `() => undefined` with the noop used in `use-query` (extract to src/utils if needed)